docs/FAQ.md

```diff
@@ -137,6 +137,20 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I resume working on an existing change?
+
+There are two ways to resume working on an earlier change: `jj new` then `jj squash`,
+and `jj edit`. The first is the recommended default, but `jj edit` can be useful.
+When you use `jj edit`, you don't get the opportunity to review your draft before it
+is committed to the revision. This can create oversight issues when drafting over
+changes that were considered done. You should avoid using `jj edit` when the
+revision has a conflict, as the state will have plain-text conflict annotations.
+
+To start, use `jj new <rev>` to create a change based on that earlier revision. Make
+your edits, then use `jj squash` to update the earlier revision with those edits.
+For when you would use git stashing, use `jj edit <rev>` for expected behaviour. 
+Other workflows may prefer `jj edit` as well.
```

Comment:
This seems a bit too vague to be useful. I would leave it out. Alternatively, describe what those workflows are (but maybe under a different heading, since this heading is about "resume working on a change")

Reply:
The point of this bit was to highlight that individual workflows could diverge to make `jj edit` more ideal as a default